pages/dkp/kommander/2.1/major-upgrade/upgrade-clusters/migrate-platform-apps/cleanup/index.md

```diff
@@ -32,42 +31,23 @@ release "elasticsearch-kubeaddons" uninstalled
 
 ## Istio
 
-If Istio is installed on the cluster, the upgrade leaves the Helm Secret
-that belonged to the 1.8 Istio Addon in the `istio-system` namespace.
+If Istio is installed on the cluster, the upgrade leaves the Helm Secret that belonged to the 1.8 Istio addon in the `istio-system` namespace.
 
-**WARNING:** As long as this secret exists, `helm list -n istio-system`
-will incorrectly report the existence of a Helm release called "istio-kubeaddons".
-If you manipulate this "Helm release" using Helm, it may corrupt the Istio
-Platform Application in Kommander, potentially disrupting operation of
-Istio workloads.
+<p class="message--note"><strong>WARNING: </strong>As long as this secret exists, <code>helm list -n istio-system</code> will incorrectly report existence of an "istio-kubeaddons" Helm release. If you manipulate this "Helm release" using Helm, it may corrupt the Istio Platform Application in Kommander, which can disrupt operation of Istio workloads.</p>
 
-Delete this secret using `kubectl`:
+Delete this secret using the `kubectl` command:
 
 ```sh
 kubectl delete -n istio-system secret sh.helm.release.v1.istio-kubeaddons.v1
 ```
 
-<p class="message--note"><strong>WARNING: </strong>
-As long as this secret exists, <code>helm list -n istio-system</code>
-will wrongly report existence of a Helm release called "istio-kubeaddons".
-If you manipulate this "Helm release" using Helm, it may corrupt the Istio
-Platform Application in Kommander, potentially disrupting operation of
-Istio workloads.
-</p>
-
 ## Gatekeeper
 
-If Gatekeeper is installed on the cluster, the upgrade leaves a Helm Secret,
-Deployments, and a Service that belonged to the 1.8 Gatekeeper Addon
-in the `kubeaddons` namespace.
+If Gatekeeper is installed on the cluster, the upgrade leaves a Helm Secret, Deployments, and a Service that belonged to the 1.8 Gatekeeper addon in the `kubeaddons` namespace.
 
-**WARNING:** As long as the secret exists, `helm list -n kubeaddons`
-will incorrectly report the existence of a Helm release called "gatekeeper-kubeaddons".
-If you manipulate this "Helm release" using Helm, it may corrupt the Gatekeeper
-Platform Application in Kommander, potentially making Kubernetes on the cluster
-inoperable.
+<p class="message--note"><strong>WARNING: </strong>As long as the secret exists, `helm list -n kubeaddons` will incorrectly report the existence of a "gatekeeper-kubeaddons" Helm release. If you manipulate this "Helm release" using Helm, it may corrupt the Gatekeeper Platform Application in Kommander, potentially making Kubernetes on the cluster inoperable.</p>
```

Comment:
```suggestion
<p class="message--note"><strong>WARNING: </strong>As long as the secret exists, `helm list -n kubeaddons` incorrectly reports the existence of a "gatekeeper-kubeaddons" Helm release. If you manipulate this "Helm release" using Helm, it might corrupt the Gatekeeper Platform Application in Kommander, potentially making Kubernetes on the cluster inoperable.</p>
```